administration/src/mui-modules/application/primitive-inputs/EmailForm.tsx

```diff
@@ -22,6 +23,12 @@ const EmailForm: Form<State, ValidatedInput, AdditionalProps> = {
         message: `Text überschreitet die maximal erlaubten ${MAX_SHORT_TEXT_LENGTH} Zeichen.`,
       }
     }
+    if (!isEmailValid(email)) {
+      return {
+        type: 'error',
+        message: `Ungültige E-Mail.`,
```

Comment:
```suggestion
        message: `E-Mailadresse ist ungültig.`,
```
Since we use proper sentences in the messages before we should keep this

Reply:
The Duden dictionary recommends writing the term as "E-Mail-Adresse" (see https://www.duden.de/rechtschreibung/E_Mail_Adresse). The rule for this is that foreign words and German words are joined with a hyphen.